### ISSUE TITLE
`AiohttpClient.__str__()` references missing instance attribute `self.pool_size`

### ISSUE THREAD

**Issue by JakeUrban** [StellarCN/py-stellar-base#510]
**Describe the bug**

The `AiohttpClient.__str__()` method references a missing instance attribute, `self.pool_size`. `pool_size` is passed as a parameter to the constructor but is not persisted on the object.

**What version of dependencies are you using?**

v4.1

**To Reproduce**

```
>>> from stellar_sdk.client.aiohttp_client import AiohttpClient
>>> print(AiohttpClient())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../stellar_sdk/client/aiohttp_client.py", line 237, in __str__
    f"<AiohttpClient [pool_size={self.pool_size}, "
AttributeError: 'AiohttpClient' object has no attribute 'pool_size'
```

**Expected behavior**

Either save `self.pool_size` in `__init__()` or remove the attribute from the string representation.

**Additional context**

n/a

**Comment by overcat** [comment]
Sorry for the inconvenience, it has been fixed in [4.1.1](https://github.com/StellarCN/py-stellar-base/releases/tag/4.1.1).

**Comment by JakeUrban** [comment]
Thanks! It helps when debugging tests, since the Server and Client classes need to be mocked.
